src/components/Icon/index.tsx

```diff
@@ -103,3 +103,4 @@ class Icon extends PureComponent<IconProps> {
 }
 
 export default withTheme(withThemeStyles(withStyleUtils(Icon)));
+export type {SrcProps};
```

Comment:
Can we move this type to the `type.ts`?

Reply:
@pasyukevich Which one? Or do you mean to create a new one? Usually, we create` types.ts` to share the types between multi-platform implementation

Reply:
Yeah, I mean to create a new one, since this type is used outside also

Reply:
Since we have other places with the same approach, I think we can leave it as is, at least for now.